src/librustc/mir/interpret/mod.rs

```diff
@@ -187,19 +187,18 @@ enum State {
 }
 
 pub struct AllocDecodingState {
-    // For each AllocId we keep track of which decoding state it's currently in.
-    decoding_state: Vec<Mutex<State>>,
+    // For each `AllocId`, we keep track of which decoding state it's currently in.
+    decoding_state: Vec<Lock<State>>,
```

Comment:
I have no context for reviewing this semantic change.

Reply:
It's not semantic actually; notice I just removed the name alias for the import. ;-) It's now more consistent with other modules in the same location, more or less.